core/arch/x86/decode_table.c

```diff
@@ -1781,6 +1790,12 @@ const instr_info_t * const op_instr[] =
 #define Me TYPE_M, OPSZ_16_vex32_evex64
 #define Ue TYPE_V_MODRM, OPSZ_16_vex32_evex64
 
+/* MPX additions, own codes */
+#define TRq_dq TYPE_T_REG, OPSZ_8_dimode16
```

Comment:
Underscore means a partial size out of a larger size, which this is not.  Not sure what is best: doesn't seem worth coming up with a new letter.  If we had different case I would put an x in between: `qXdq`.  Maybe just run them together: `qdq`.

Reply:
TRq_dq => TRqdq.
TMq_dq => TMqdq.